src/client/dfuse/SConscript

```diff
@@ -62,13 +63,29 @@ def build_client_libs_shared(env, prereqs):
     il_obj = []
     for src in IOIL_SRC:
         il_obj += ilenv.SharedObject(os.path.join('il', src), SHOBJPREFIX='s_')
+
+    pil4dfsenv = env.Clone()
+    pil4dfsenv.AppendUnique(CFLAGS='-Wno-error=nonnull-compare')
+    if '-fvisibility=hidden' in pil4dfsenv['CFLAGS']:
```

Comment:
Why do we need to remove this?  Seems like it would be better to annotate public functions with DFUSE_PUBLIC?

Reply:
Eventually, it might be possible that this library does not need dfuse. I did not use dfuse related code in this library unless really necessary.

Reply:
I used "static" for all other functions are not supposed to be exposed. Removing "-fvisibility=hidden" should be fine.